fachhochschule-vorarlberg.csl

```diff
@@ -101,7 +145,8 @@
       <else-if type="article-journal article-newspaper article-magazine entry-encyclopedia" match="any">
         <choose>
           <if variable="author">
-            <text variable="title" font-style="normal" prefix="„" suffix="“"/>
+            <!--suffix um . vor dem Hochkomma erweitert! lt. kuma (zeitschriftenartikel)-->
+            <text variable="title" font-style="normal" prefix="„" suffix=".“"/>
```

Comment:
is there any reason we're not doing these with quotes="true" instead of the hardcocded quotation marks?